internal/server/data/data.go

```diff
@@ -104,6 +104,10 @@ func get[T models.Modelable](db *gorm.DB, selectors ...SelectorFunc) (*T, error)
 		db = selector(db)
 	}
 
+	if hasOrgID(new(T)) {
+		db = ByOrgID(OrgFromContext(db.Statement.Context).ID)(db)
```

Comment:
Should we just default to always setting this org ID for the time being? I can't think of an example of something that shouldn't have an org ID.

Reply:
Ah, of course, organizations themselves. 👍

Reply:
some tables don't have an OrganizationID column